exercises/practice/ari/test-ari.bats

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bats
+load bats-extra
+
+setup() {
+
+    cat > exercism.txt <<EOF
```

Comment:
@glennj  Do we need temp files? If we do, can we use `mktemp`? If we don't, can we just use functions that generate the text? Thoughts?